frontend/src/app/patients/EditPatient.tsx

```diff
@@ -313,11 +321,11 @@ const EditPatient = (props: Props) => {
         </div>
       </div>
       <div className="display-flex flex-align-center">
-        {!props.fromQueue && (
+        {!props.fromQueue && formChanged ? (
```

Comment:
The logic here is causing a "Start test" button to appear if we click a patient from the "Conduct tests" page

https://github.com/CDCgov/prime-simplereport/assets/20211771/9658f8d0-0c5b-46b0-82c8-ee3103099d76

The "Save changes" button should also be disabled since we haven't made any changes

Reply:
ayayay I didn't think about this user flow at all -- thanks for catching this!